pkg/sql/alter_type.go

```diff
@@ -157,6 +156,44 @@ func (p *planner) performRenameTypeDesc(
 	return p.txn.Run(ctx, b)
 }
 
+func (p *planner) renameTypeValue(params runParams, n *alterTypeNode, oldVal string, newVal string) error {
+	oldNameFound := false
+	newNameFound := false
+
+	var enumMemberIndex int
+
+	// Do one pass to verify that the oldVal exists and there isn't already
+	// a member that is named newVal.
+	for i, t := range n.desc.GetEnumMembers() {
+		if t.LogicalRepresentation == oldVal {
+			oldNameFound = true
+			enumMemberIndex = i
+		} else if t.LogicalRepresentation == newVal {
+			newNameFound = true
+		}
+	}
+
+	if !oldNameFound {
+		return errors.Newf("%s is not an existing enum label", oldVal)
```

Comment:
please use `pgerror`s and double check against postgres that you return the same error code and message